src/types.rs

```diff
@@ -4,6 +4,12 @@
 /// A user ID.
 pub type UserId = String;
 
+/// A reward ID.
+pub type RewardId = String;
+
+/// A reward redemption ID.
+pub type RedemptionId = String;
```

Comment:
Should be used in https://github.com/Emilgardis/twitch_api2/blob/f93c1e2416de16bd1c2405c7ce4a4a2187677fd9/src/pubsub/channel_points.rs